src/test/java/com/hotels/mutantswarm/report/LineFactoryTest.java

```diff
@@ -78,41 +78,18 @@ public void setup() {
     when(statement2.getIndex()).thenReturn(0);
 
     // 'a' -> 'null'
-    when(mutation1.getReplacementText()).thenReturn("null");
```

Comment:
Lot of things removed in here, just wondering the reason ?

Reply:
I answered below